src/landscape/components/LandscapeForm/New.js

```diff
@@ -124,6 +125,16 @@ const LandscapeNew = () => {
           boundaryOption: updatedLandscape.boundaryOption,
         },
       });
+      if (updatedLandscape.boundaryOption) {
+        trackEvent('Create landscape boundary', {
+          props: {
+            landscapeName: updatedLandscape.name,
+            country: updatedLandscape.location,
+            boundaryOption: updatedLandscape.boundaryOption,
+            [ILM_OUTPUT_PROP]: MAP_LANDSCAPE_BOUNDARIES,
```

Comment:
question: is this supposed to be an array?

Reply:
@paulschreiber It is not an array it is a way in javascript to set a dynamic value for the key